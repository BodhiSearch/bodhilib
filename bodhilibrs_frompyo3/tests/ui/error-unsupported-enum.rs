use frompyo3::FromPyO3;

#[derive(FromPyO3)]
enum TestUnsupportedEnum {}

fn main() {}
