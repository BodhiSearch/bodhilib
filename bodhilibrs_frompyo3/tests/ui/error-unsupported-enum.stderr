error: Enums are not supported
 --> tests/ui/error-unsupported-enum.rs:4:1
  |
4 | enum TestUnsupportedEnum {}
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^
